conf/messages.cy

```diff
@@ -111,7 +111,7 @@ error.no_entry_sa-postcode                                  = Nodwch y cod post
 
 ## Overseas Tax Identifier Page
 tax-identifier.title                                        = Oes gan y busnes ddynodydd treth tramor?
-tax-identifier.hint                                         = Gallai hyn ein helpu i wybod pa fusnes sydd dan sylw. Gallai hyn fod yn rhif cofrestru TAW, Rhif Adnabod y Cyflogai (EIN) neu unrhyw ddynodydd arall y gallem ei ddefnyddio i gadarnhau manylion y busnes.
+tax-identifier.line_1                                       = Efallai y byddwn yn defnyddio’r rhif hwn i’n helpu i ddod o hyd i’ch busnes. Gallai’r dynodydd treth fod yn rhif cofrestru TAW, Rhif Adnabod y Cyflogai (EIN) neu unrhyw ddynodydd arall y gallem ei ddefnyddio er mwyn gwirio manylion eich busnes.
```

Comment:
Guess when you reverted the branch, message got reverted to old message, please check with the message content mentioned in the ticket